### PR TITLE
languages update; fields for gam info user; cloud identity groups update to v1

### PR DESCRIPTION
**Pull request by taers232c**
The API doesn't return languages unless you specifically mention in in fields list

### CODE REVIEW

File: src/gam/__init__.py [hunk 1]
```diff
@@ -9814,6 +9840,9 @@ def doPrintUsers():
                     if user_email.find('@') != -1:
                         user['primaryEmailLocal'], user[
                             'primaryEmailDomain'] = splitEmailAddress(user_email)
+                up = 'languages'
+                if up in user:
```
Comment (jay0lee) [inline src/gam/__init__.py:9844]: No reason to keep up here.

File: src/gam/__init__.py [hunk 2]
```diff
@@ -6703,6 +6703,7 @@ def _splitSchemaNameDotFieldName(sn_fn, fnRequired=True):
                 if language.lower() in LANGUAGE_CODES_MAP:
                     lang_item['languageCode'] = LANGUAGE_CODES_MAP[language.lower()]
                 else:
+                    lang_item.pop('preference', None)
```
Comment (jay0lee) [inline src/gam/__init__.py:6706]: this should not be necessary as preference is not set if customLanguage is set:

https://github.com/jay0lee/google-api-tracker/blob/master/admin-directory_v1.json#L7367

**Comment by taers232c** [comment]
The user can say: gam update user billybob languages drawl+,en-US-
preference is set from +- before the language is detected as custom
so we clear it if set on a custom language
--
***@***.***

**Comment by jay0lee** [comment]
I'd prefer to see that throw an API error (which the user can then correct) as opposed to making assumptions about what the user meant to enter. Suppose user mis-typed a non-customer language?

**Pull request merged**